json/src/main.cpp

```diff
@@ -325,11 +325,25 @@ namespace json{
 
         return v;
     }
+
+
+    Value len(std::vector<Value>& args, Ark::VM *vm){
+        /*
+            args[0] -> json_object
+        */
+
+        if (args.size() != 1)
+            throw std::runtime_error("json:len: needs 1 argument: jsonAsList or jsonAsObject");
+        if (args[0].usertype_ref().as<nlohmann::json>().is_array() || args[0].usertype_ref().as<nlohmann::json>().is_object())
+            return Value(static_cast<int>(args[0].usertype_ref().as<nlohmann::json>().size()));
```

Comment:
cast it as a long instead of int to avoid losing precision. underneath it's converted to a double, but a double can store as much as a long can so we'll be fine